.github/workflows/integration-test.yml

```diff
@@ -0,0 +1,35 @@
+name : Integration Tests
+on :
+  # allow manual triggering 
+  workflow_dispatch:
+  
+  # for debugging
+  push:
+  pull_request:
+
+  schedule:
+    # run Thursday 4:30 AM UTC
+  - cron: '30 4 * * 4'    
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    env:
+      GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}
+    
+    container: 
+      image: pecan/base:develop
+
+    steps:
+    - name: Checkout source code
+      uses: actions/checkout@v3
+
+    - name: Install mockery
+      run: |
+        Rscript -e "install.packages('mockery', repos='https://ftp.belnet.be/mirror/CRAN')"
```

Comment:
I don't /think/ this is what's causing the CI failure for this PR, but is this mirror known to be reliable and up-to-date? In general I recommend leaving `repos` unset wherever unless there's a specific reason to override it, so that all installations use the mirror(s) configured by the base image.

Reply:
Actually without this as well the CI was throwing the same error, thus on a trial basis I added this to test if this might solve the issue. Thus will be removing this out anyway.